docs/Data-Loading.md

```diff
@@ -27,6 +27,11 @@ If you want the scripts to be able to generate html reports (recommended way for
 $ sudo python3 -m pip install Jinja2
 ```
 
+To be able to display the [study metadata](File-Formats.md##study-metadata-file), you need the module `pyyaml`. You can use this command to install the module:
```

Comment:
Please rephrase. `pyyaml` is now needed for every validation, if it contains metadata or not, right?